tests/test_shape.py

```diff
@@ -28,3 +29,23 @@ def test_polygon_not_empty_np_array():
     geom = {"type": "Polygon", "coordinates": np.array([[[5, 10], [10, 10], [10, 5]]])}
     obj = shape(geom)
     assert obj == Polygon([(5, 10), (10, 10), (10, 5)])
+
+
+@pytest.mark.parametrize("geom", [
+    {'type': "MultiLineString", "coordinates": []},
+    {'type': 'MultiLineString', 'coordinates': [[]]},
+    {'type': 'MultiLineString', 'coordinates': None}
+])
+def test_multilinestring_empty(geom):
+    assert shape(geom) == MultiLineString()
```

Comment:
Should those tests also be added to the master branch? (I understand that this PR targetted maint-1.7, as it was already fixed on master)

Reply:
Merged maint-1.7 in 73f4ccb.